server/api.js

```diff
@@ -777,15 +777,16 @@ router.deleteAsync("/lobby", ensure.isAdmin, async (req, res) => {
  * Add self as a referee to a quals lobby
```

Comment:
nit: update comment for this and lobby-player

Reply:
done